packages/bundle-source/cache.js

```diff
@@ -69,7 +70,6 @@ export const makeBundleCache = (wr, cwd, readPowers, opts) => {
     // Prevent other processes from doing too much work just to see that we're
     // already on it.
     await metaWr.rm({ force: true });
-    await bundleWr.rm({ force: true });
```

Comment:
Did this change obviate the need to address PID collisions?

Reply:
IOW, I think we still need to account for intra-process collisions.

Reply:
This might suffice https://github.com/endojs/endo/pull/1760

It might make sense to pull the now-optional `pid` argument from the maker. But, it might also make sense to keep the PID (so another tool can clean up dead scratch detritus) and add a nonce to the scratch file names. I’ll take that conversation there.

Reply:
I think we should delete the previous `metaWr.rm` as well. These are dross from the revision before atomic renames and in the new model, removing the old files is not necessary.